src/en/wip-storage.md

```diff
@@ -107,29 +120,45 @@ Storage requirements _may_ be added to the 'metadata.yaml' of the charm as follo
 storage:
   data:
     type: filesystem
+    description: junk storage
+    shared: false
```

Comment:
if we add shared and read-only can we add a note here that they're not supported yet